machine/v1beta1/0000_10_machine.crd.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
+    capability.openshift.io/name: MachineAPI+baremetal
```

Comment:
[The `+` syntax][1] means that this `Machine` CRD will only be managed when both `MachineAPI` and `baremetal` are enabled.  But it's not obvious to me why we wouldn't want `Machine` when `MachineAPI` alone was enabled  and `baremetal` was not.  I don't see anything in the commit message or pull request comments explaining how `baremetal` is involved.  What is the context?

[1]: https://github.com/openshift/enhancements/blame/9e6923d2c650aa9efce2f6eedd97bee345427080/enhancements/installer/component-selection.md#L142-L144

Reply:
Is there an `or` syntax for the capabilities?

Reply:
The lack of `or` syntax is intentional, [because][1]:

> Our current plan is to only annotate resources that are not depended upon (aka "leaf" components)

Is there a description somewhere about what the `baremetal` component enables when `MachineAPI` is disabled?  There's not enough detail in [the `baremetal` docstring][2] for me to understand what overlaps with `MachineAPI` and what is distinct from `MachineAPI`.

[1]: https://github.com/openshift/enhancements/blame/9e6923d2c650aa9efce2f6eedd97bee345427080/enhancements/installer/component-selection.md#L357
[2]: https://github.com/openshift/api/blob/ea40115b9fa67188f89075ec4e34a9ea346ea123/config/v1/types_cluster_version.go#L261-L264